### ISSUE TITLE
need 2 optimizers?

### ISSUE THREAD

**Issue by cheolhwanyoo** [anita-hu/MSAF#5]
Hello.
There are 2 optimizers in the training phase, and it seems to learn the pose stream and apperance stream alternately.
Is there a reason why you didn't learn to cross-entropy loss using only one optimizer after combining the results of the pose stream and appearance stream?
Thank you for always.

**Comment by anita-hu** [comment]
Since we use the same configuration for both optimizers, it is equivalent to use one. But multiple optimizers are useful when you want a different configuration for each stream, for example, freezing parameters, different learning rates, different optimization algorithms, etc.
